digitalstrategie/assets/scss/components/_ds-block-overwrites.scss

```diff
@@ -1,15 +1,15 @@
 // overwrite spacing of #layout-grid__area--maincontent
-.ds-block__maincontent {
-    min-height: 0 !important;
-    margin-top: 0 !important;
-    margin-bottom: 0 !important;
-}
+// .ds-block__maincontent {
+//     min-height: 0 !important;
+//     margin-top: 0 !important;
+//     margin-bottom: 0 !important;
+// }
```

Comment:
I don'T think we need this anymore

Reply:
No, that was just for overwriting when I used their grid all wrong. You can remove it!